tests/boards/ehl_crb/gpio/prj.conf

```diff
@@ -0,0 +1,5 @@
+# nothing here
```

Comment:
please remove this comment line (I don't see this in other prj.conf, also it seems unnecessary and does not seem to add value)